LTRegistratorApi/LTRegistratorApi/LTRegistratorApi.csproj

```diff
@@ -10,8 +10,15 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNet.Identity.Core" Version="2.2.2" />
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.1.2" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="2.1.11">
```

Comment:
I am not sure that it is really needed. According to "ASP .NET core in Action" everything is already included in Microsoft.AspNetCore.App. Please try to remove them and start project.